pkgs/applications/science/physics/xfitter/undefined_behavior.patch

```diff
@@ -1,5 +1,4 @@
 diff --git a/DY/src/finterface.cc b/DY/src/finterface.cc
-index 0405786..eb171d0 100644
 --- a/DY/src/finterface.cc
```

Comment:
Have you also send this patch upstream?

Reply:
Did you had a chance to submit the patch?

Reply:
@Mic92  I did. Fingers crossed, will see how it goes.